src/Bus.js

```diff
@@ -105,11 +105,11 @@ class Bus {
             this._mozaik.logger.info(`Setting timer for "${ requestId }"`);
             this.subscriptions[requestId].timer = setInterval(() => {
                 this.processApiCall(requestId, callFn, request.params);
-            }, 10000);
+            }, 100000);
```

Comment:
This seems to modify the timer interval from 10 to 100 seconds - thoughts on this change?